### PR TITLE
Privacy 2024 Chapter

### LINKED ISSUE #3603: Privacy 2024
# Privacy 2024

Privacy illustration

If you're interested in contributing to the Privacy chapter of the 2024 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide). You might be interested in exploring the changes to this year's version [here](https://github.com/HTTPArchive/almanac.httparchive.org/discussions/3619).

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| ---- | ------- | --------- | -------- | ------- |    -------  |
| @UmarIqbal  | @umariqbal, @ChrisBeeti,  @shaoormunir, @turban1988, @ydimova     | @lord-r3, @AlbertoFDR | @Yash-Vekaria, @max-ostapenko, @bstandaert-wustl | @cqueern  | @turban1988   | 

<details>
<summary><b>Expand for more information about each role 👀</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [0. Form the content team](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
- [x] 📆 `April 15` Complete program and content committee - 🔑 *Organizing committee*
  - The content team has at least one author, reviewer, and analyst.

### [1. Plan content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
- [x] 📆 `May 1` **First meeting to outline the chapter contents** - 🔑 *Content team*
  - The content team has completed the chapter outline.

### [2. Gather data](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
- [x] 📆 `June 1` **Custom metrics completed** - 🔑 *Analysts*
  - Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/custom-metrics/blob/main/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress. 
- [x]  📆 `June 1` **HTTP Archive Crawl** - 🔑 *HA Team*
  - HTTP Archive runs the June crawl.

### [3. Validate results](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
- [x] 📆 `August 15` **Query Metrics & Save Results** - 🔑 *Analysts*
  - Analysts have queried all metrics and saved the output.

### [4. Draft content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
- [x] 📆 `September 15` **First Draft of Chapter** - 🔑 *Authors*
  - Authors has written the chapter. 
- [x] 📆 `October 10` **Review & Edit Chapter** - 🔑 *Reviewers & Editors*
  - Reviewers and  Editors has processed the the chapter.

### [5. Publication](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
- [ ] 📆 `October 15` **Chapter Publication (Markdown & PR)** - 🔑 *Authors*
  - Authors has converted the chapter to markdown and drafted a PR.
- [ ] 📆 `November 1` **Launch of 2024 Web Almanac 🚀** - 🔑 *Organizing committee*
 
### [6. Virtual conference](#)
- [ ] 📆 `November 20` **Virtual Conference** - 🔑 *Content Team*

<a id="chapter-resources"></a>
## [Chapter resources](#chapter-resources)
Refer to these 2024 Privacy resources throughout the content creation process:
[📄 Google Docs](https://docs.google.com/document/d/14XXHV7A8WI1a-UQqmlRj4dKYIJjZMqo__CN5POfM9Po/edit) for outlining and drafting content
[🔍 SQL files](https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2024/privacy/README.md) for committing the queries used during analysis
[📊 Google Sheets](https://docs.google.com/spreadsheets/d/18r8cT6x9lPdM-rXvXjsqx84W7ZDdTDYGD59xr0UGOwg/edit#gid=1778117656) for saving the results of queries
[📝 Markdown file](https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en/2023/privacy.md) for publishing content and managing public metadata
[💻 Collab notebook](https://colab.research.google.com/drive/1n9CeYIt3-Rtzvu1o6khn0KKc4I7ofkQK) for collaborative coding in Python - if needed
[💬 #web-almanac-privacy](https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA) on Slack for team coordination

### PR DESCRIPTION
**Pull request by max-ostapenko**
Fixes #3603 

Staged link: https://privacy-2024-dot-webalmanac.uk.r.appspot.com/en/2024/privacy
Staged featured chapter home page link: https://privacy-2024-dot-webalmanac.uk.r.appspot.com/en/2024/?feat=privacy

- [x] Add chapter metadata (contributors, bios, featured)
- [x] Add markdown code and review it
- [x] Add figures markdowns

### CODE REVIEW

**Comment by tunetheweb** [comment]
Can we merge this with #3813 ?

File: src/content/en/2024/privacy.md [hunk 1]
```diff
@@ -16,3 +16,85 @@ featured_stat_label_2:
...
+  sql_file="number_of_websites_with_whotracksme_trackers.sql"
+  )
+}}
+
+### Third-Party Cookies
+
+{{ figure_markup(
+  image="top-third-party-cookie-origins.png",
+  caption="Top Third-Party Cookie Origins.",
+  description="A bar chart showing the most-common origins for third-party cookies, by the percent of pages on which they appear. Displayed values are doubleclick.net (27% desktop; 26% mobile), youtube.com (7% desktop; 6% mobile), google.com (5% desktop; 4% mobile), www.google.com (5% desktop; 4% mobile), linkedin.com (4% desktop; 4% mobile), bing.com (4% desktop; 3% mobile), yandex.ru (3% desktop; 5% mobile), adnxs.com (3% desktop; 3% mobile), mc.yandex.ru (3% desktop; 4% mobile), c.bing.com (3% desktop; 3% mobile), yandex.com (3% desktop; 4% mobile), mc.yandex.com (3% desktop; 4% mobile), adsrvr.org (3% desktop; 3% mobile), googleadservices.com (3% desktop; 3% mobile), yahoo.com (3% desktop; 3% mobile)."
```
Comment (max-ostapenko) [inline src/content/en/2024/privacy.md:45]:
```suggestion
  description="A bar chart showing the most-common origins for third-party cookies, by the percent of pages on which they appear. Displayed values are doubleclick.net (27% desktop; 26% mobile), youtube.com (7% desktop; 6% mobile), google.com (5% desktop; 4% mobile), `www.google.com` (5% desktop; 4% mobile), linkedin.com (4% desktop; 4% mobile), bing.com (4% desktop; 3% mobile), yandex.ru (3% desktop; 5% mobile), adnxs.com (3% desktop; 3% mobile), mc.yandex.ru (3% desktop; 4% mobile), c.bing.com (3% desktop; 3% mobile), yandex.com (3% desktop; 4% mobile), mc.yandex.com (3% desktop; 4% mobile), adsrvr.org (3% desktop; 3% mobile), googleadservices.com (3% desktop; 3% mobile), yahoo.com (3% desktop; 3% mobile)."
```
@bstandaert-wustl 
This one is recognised as URL in the chart legend and gives an error during html conversion.

Comment (max-ostapenko) [inline src/content/en/2024/privacy.md:45]:
```suggestion
  description="A bar chart showing the most-common origins for third-party cookies, by the percent of pages on which they appear. Displayed values are doubleclick.net (27% desktop; 26% mobile), youtube.com (7% desktop; 6% mobile), google.com (5% desktop; 4% mobile), www.google.com (5% desktop; 4% mobile), linkedin.com (4% desktop; 4% mobile), bing.com (4% desktop; 3% mobile), yandex.ru (3% desktop; 5% mobile), adnxs.com (3% desktop; 3% mobile), mc.yandex.ru (3% desktop; 4% mobile), c.bing.com (3% desktop; 3% mobile), yandex.com (3% desktop; 4% mobile), mc.yandex.com (3% desktop; 4% mobile), adsrvr.org (3% desktop; 3% mobile), googleadservices.com (3% desktop; 3% mobile), yahoo.com (3% desktop; 3% mobile).",
```
@bstandaert-wustl missing commas in every chart markup

**Comment by max-ostapenko** [comment]
@tunetheweb is website redirection test failing an issue in the CI?

**Comment by tunetheweb** [comment]
Oh it's the featured chapter bit that's failing. this fixes it:

```
featured_quote: TODO
featured_stat_1: TODO
featured_stat_label_1: TODO
featured_stat_2: TODO
featured_stat_label_2: TODO
featured_stat_3: TODO
featured_stat_label_3: TODO
```

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**